playbooks/pulp/pre.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: all
+  tasks:
+    - name: Ensure git is installed
```

Comment:
@pabelanger Can we assume this is always the case ?

Reply:
no, it is safer to add it into bindep.txt file in your project repo, then we'll ensure it is installed